src/Nri/Ui/Carousel/V1.elm

```diff
@@ -0,0 +1,94 @@
+module Nri.Ui.Carousel.V1 exposing (..)
+
+import Css exposing (..)
+import Html.Styled as Html exposing (Html)
+import Nri.Ui
+import Nri.Ui.Tooltip.V3 as Tooltip
+import TabsInternal.V2 as TabsInternal
+
+
+type Tab id msg
+    = Tab (TabsInternal.Tab id msg)
+
+
+type Attribute id msg
+    = Attribute (TabsInternal.Tab id msg -> TabsInternal.Tab id msg)
+
+
+type TabPosition
```

Comment:
Are you expecting to need both position styles? Or do you think the controls are likely always appear below the tabpanel?

Generally speaking, I'd rather not support use-cases we haven't encountered yet 🤔

Reply:
I think we can follow this: https://github.com/NoRedInk/noredink-ui/pull/951#discussion_r898583775
And remove this type 😊